EasyCommands/Program.cs

```diff
@@ -132,6 +132,8 @@ public partial class Program : MyGridProgram {
 
                 if (!ParseCommands()) {
                     Runtime.UpdateFrequency = updateFrequency;
+                    state = ProgramState.RUNNING;
+                    QueueRequest(argument);
```

Comment:
What happens if `ParseCommand` has not finished parsing yet?

Reply:
If so, then the passed in request gets queued for execution and will be executed first (or, in queued order) once parsing is complete.